.jshintrc

```diff
@@ -0,0 +1,23 @@
+{
+    "browser": true,
+    "jquery": true,
+    "node": true,
+
+    "bitwise": true,
+    "curly": false,
+    "eqeqeq": true,
+    "forin": true,
+    "freeze": true,
+    "indent": 4,
+    "noarg": true,
+    "noempty": true,
+    "nonbsp": true,
+    "quotmark": "single",
+    "strict": true,
+    "undef": true,
+    "unused": "vars",
+
+    "predef": [
+        "dashboard"
+    ]
+}
```

Comment:
In fact, because I'm a jerk, here's your .eslintrc file:
```yaml
env:
  browser: true,
  jquery: true,
  node: true

globals:
  dashboard: false

rules:
  curly: 0
  eqeqeq: 2
  guard-for-in: 2
  indent: [2, 4, {SwitchCase: 1}]
  no-bitwise: 2
  no-caller: 2
  no-empty: 2
  no-extend-native: 2
  no-irregular-whitespace: 2
  no-undef: 2
  no-unused-vars: 2
  quotes: [2, single]
  strict: [2, function]
```

And if you run `$ eslint .` (it ignores ./node_modules/ by default), here is your output:
```bash
$ eslint .

/Users/pdehaan/dev/github/push-dev-dashboard/dashboard/static/js/analytics.js
  153:21  error  Expected indentation of 16 space characters but found 20  indent

/Users/pdehaan/dev/github/push-dev-dashboard/gulpfile.js
  1:1   error  Use the function form of "use strict"  strict
  8:26  error  Use the function form of "use strict"  strict

✖ 3 problems (3 errors, 0 warnings)
```

So, it caught some bad indentation, and is a bit more strict about global `strict` usage.
We can ignore both the `Use the function form of "use strict"` errors since we can remove that gulpfile.js and use ESLint directly.